_posts/2018-06-28-foreman-community-newsletter---june-2018.md

```diff
@@ -0,0 +1,198 @@
+the CFP for presentations closes *this week* (SUnday 1st) - so if you want to
+get your entry paid for, get your submissions in right away!
+
+#### [Past Events](https://community.theforeman.org/c/events/l/latest)
+
+* [OpenSourceCamp.de](https://community.theforeman.org/t/open-source-camp-berlin-14-june/8738)
+
+Our first ever Foreman-headlined conference was held earlier this month, and
+was great fun - thanks to our partners at NETWAYS and Graylog for helping to
+make this event happen. A full day of talks, hacking, and discussion,
+thoroughly enjoyed by all (so I am told :P)
+
+* [Community Demo - 31st May](https://community.theforeman.org/t/foreman-community-demo-45/9698)
+
+Containers, backups, and oVirt, oh my! A return to force for the demo, with
+great content from all the presenters.
+
+* [Community Demo - 21st June](https://community.theforeman.org/t/foreman-community-demo-46/9732)
+
+A packed demo, covering the Redmine and Survey results mentioned above, as well
+as foreman-hooks for content, more Docker support, better state handling for
+unattended installs (awesome!) and more. One to watch.
+
+* [UX Dive on the Audits page](https://community.theforeman.org/t/foreman-usability-audits-ui/10093)
+
+Tereza Novotna took us through the proposed wireframes for the old Audits UI,
+which has been in need of love for some time. We also had time for a solid
+discussion around filtering vs sorting, and tables vs list views.
+
+### Plugin news
+
+Updated plugins:
+- [hammer_cli_foreman_remote_execution](https://github.com/theforeman/hammer_cli_foreman_remote_execution) updated to 0.1.0
+- [foreman_chef](https://github.com/theforeman/foreman_chef) updated to 0.8.0
+- [foreman_tasks](https://github.com/theforeman/foreman_tasks) updated to 0.13.2
+- [foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution)to 1.5.3
+- [foreman_discovery](https://github.com/theforeman/foreman_discovery) updated to 12.0.1
+- [foreman_docker](https://github.com/theforeman/foreman_docker) updated to 4.1.0
+- [foreman_openscap](https://github.com/theforeman/foreman_openscap) updated to 0.10.1
+- [hammer_cli_katello](https://github.com/Katello/hammer-cli-katello) updated to 0.13.2
+- [foreman_memcache](https://github.com/theforeman/foreman_memcache) updated to 0.1.1
+- [smart_proxy_monitoring](https://github.com/theforeman/smart_proxy_monitoring) updated to 0.1.2
+- [smart_proxy_omaha](https://github.com/theforeman/smart_proxy_omaha) updated to 0.0.4
+- [foreman_virt_who_configure](https://github.com/theforeman/foreman_virt_who_configure) updated to 0.2.2
+
+### Acknowledgements
+
+Thanks to everyone who contributes to the Foreman community - patches, plugins,
+testing, bug reports, translations, interviews, presentations, meetups and
+everything else. Your efforts are very much appreciated!
+
+[1_18_rc1]: https://community.theforeman.org/t/1-18-0-rc1-has-been-released/9987
```

Comment:
Interesting, I didn't know you could do this with markdown (point to these links)

Reply:
yeah, its particularly handy for making a bibliography of common links at the bottom of template posts :)